docs/source/datasets/index.rst

```diff
@@ -26,6 +26,8 @@ A **mailing address** represents the address a simulant uses to receive mail,
 which may be different -- for example, a PO box.
 Mailing addresses, not physical addresses, are recorded in tax filings.
 
+Note that in the small-scale simulated population that is available by default, these addresses all have their city/state/zip code set to the fictitious location of Anytown, WA 00000.  To read more about obtaining large-scale data with more realistic city, state, and zip code data, please see :ref:`Simulated populations <simulated_populations_main>`.
```

Comment:
Maybe we should say something about _why_ the sample population has a single city and zipcode, namely, so that linking won't be too easy.

Reply:
Good idea, but let's do that in another PR.  I slacked Caroline to ask her to add a ticket for this.